js/ui/map.js

```diff
@@ -586,7 +594,7 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
         } else if (style instanceof Style) {
             this.style = style;
         } else {
-            this.style = new Style(style, this.animationLoop);
+            this.style = new Style(style, this.animationLoop, this._workerCount);
```

Comment:
Ah! I didn't consider the `setStyle` case. I agree that this `this._workerCount` is necessary for now 👍